service/src/main/java/bio/terra/cbas/service/RunSetsService.java

```diff
@@ -243,6 +252,15 @@ private WdsRecordResponseDetails fetchWdsRecords(
         recordIdsWithError.put(recordId, e.getMessage());
       }
     }
+    micrometerMetrics.stopTimer(
+        wdsFetchRecordsSample,
+        "wds_fetch_records_timer",
+        "run_set_id",
+        runSet.runSetId().toString(),
+        "recordResponses",
+        recordResponses.toString(),
```

Comment:
Does this emit all the record details fetched to Azure Portal?

Reply:
looks like it does. do you think it'll be too heavy?

here's an example, which seems ok to me, but maybe the records are relatively simple:
```
[class RecordResponse {
id: ERR4868270
type: sra
attributes: {sample_collection_date=2020-10-31, sequencing_platform=OXFORD_NANOPORE, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/13b52460-53f7-4090-b2a9-c29b4bde26c9/call-Fetch_SRA_to_BAM/attempt-2/execution/ERR4868270.json, sra_id=ERR4868270, biosample_accession=SAMEA7612766, sequencing_platform_model=GridION, sample_strain=, sample_collected_by=Department of Pathology, University of Cambridge, library_id=COG-UK/CAMB-1B8D87/CAMB:20201106_2020_X5_FAO13851_8f36dbb7, sample_geo_loc=United Kingdom, sequencing_center=Department of Pathology, University of Cambridge, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/13b52460-53f7-4090-b2a9-c29b4bde26c9/call-Fetch_SRA_to_BAM/attempt-2/execution/ERR4868270.bam, run_date=2020-12-28}
additionalProperties: null
}, class RecordResponse {
id: SRR11953796
type: sra
attributes: {sample_collection_date=2020-03-10, sequencing_platform=ILLUMINA, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/777878c1-1976-4f9e-8b69-e60db971ce31/call-Fetch_SRA_to_BAM/execution/SRR11953796.json, sra_id=SRR11953796, biosample_accession=SAMN14938612, sequencing_platform_model=Illumina HiSeq 2500, sample_strain=not applicable, sample_collected_by=Massachusetts General Hospital, library_id=SAMN14938612_Next_2_ERCC-42, sample_geo_loc=USA: Massachusetts, sequencing_center=Broad Institute of Harvard and MIT, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/777878c1-1976-4f9e-8b69-e60db971ce31/call-Fetch_SRA_to_BAM/execution/SRR11953796.bam, run_date=2020-06-08}
additionalProperties: null
}, class RecordResponse {
id: SRR11954115
type: sra
attributes: {sample_collection_date=2020-03-10, sequencing_platform=ILLUMINA, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/a7e2656d-4be8-4f63-9aff-02258b21c38d/call-Fetch_SRA_to_BAM/execution/SRR11954115.json, sra_id=SRR11954115, biosample_accession=SAMN14938612, sequencing_platform_model=Illumina HiSeq 2500, sample_strain=not applicable, sample_collected_by=Massachusetts General Hospital, library_id=SAMN14938612_Next_1_ERCC-42, sample_geo_loc=USA: Massachusetts, sequencing_center=Broad Institute of Harvard and MIT, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/a7e2656d-4be8-4f63-9aff-02258b21c38d/call-Fetch_SRA_to_BAM/execution/SRR11954115.bam, run_date=2020-06-08}
additionalProperties: null
}, class RecordResponse {
id: SRR11954227
type: sra
attributes: {sample_collection_date=2020-03-10, sequencing_platform=ILLUMINA, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/38a79649-d5ac-4a1f-9c4e-ceb1d5e97220/call-Fetch_SRA_to_BAM/execution/SRR11954227.json, sra_id=SRR11954227, biosample_accession=SAMN14938612, sequencing_platform_model=Illumina HiSeq 2500, sample_strain=not applicable, sample_collected_by=Massachusetts General Hospital, library_id=SAMN14938612_xGen_1_ERCC-42, sample_geo_loc=USA: Massachusetts, sequencing_center=Broad Institute of Harvard and MIT, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/38a79649-d5ac-4a1f-9c4e-ceb1d5e97220/call-Fetch_SRA_to_BAM/execution/SRR11954227.bam, run_date=2020-06-08}
additionalProperties: null
}, class RecordResponse {
id: SRR13379731
type: sra
attributes: {sample_collection_date=2020-11-30, sequencing_platform=ILLUMINA, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/73972cbb-bae2-48d2-808d-71ab4e55ca41/call-Fetch_SRA_to_BAM/execution/SRR13379731.json, sra_id=SRR13379731, biosample_accession=SAMN17251688, sequencing_platform_model=NextSeq 550, sample_strain=SARS-CoV-2/USA/44165/2020, sample_collected_by=Utah Public Health Laboratory, library_id=UT-UPHL-2012556126, sample_geo_loc=USA: Utah, sequencing_center=UPHL_ID, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/73972cbb-bae2-48d2-808d-71ab4e55ca41/call-Fetch_SRA_to_BAM/execution/SRR13379731.bam, run_date=null}
additionalProperties: null
}, class RecordResponse {
id: SRR13448529
type: sra
attributes: {sample_collection_date=2020-07-09, sequencing_platform=ILLUMINA, sra_metadata=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/119ed049-cc7e-4774-b6f6-b07b5ffced0c/call-Fetch_SRA_to_BAM/execution/SRR13448529.json, sra_id=SRR13448529, biosample_accession=SAMN17358663, sequencing_platform_model=Illumina MiSeq, sample_strain=SARS-CoV-2/USA/NMDOH-2020201018/2020, sample_collected_by=New Mexico Department of Health Scientific Laboratory, library_id=NMDOH-2020201018, sample_geo_loc=USA: New Mexico, sequencing_center=New Mexico Department of Health Scientific Laboratory, reads_ubam=https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-05aa0284-81e6-45f9-a6ce-4daa2cadc017/workspace-services/cbas/terra-app-21f17b6e-d5f6-4fee-96b8-4967d4296918/fetch_sra_to_bam/119ed049-cc7e-4774-b6f6-b07b5ffced0c/call-Fetch_SRA_to_BAM/execution/SRR13448529.bam, run_date=null}
additionalProperties: null
}]
```

Reply:
For now I'm logging record IDs instead